### PR TITLE
Fixed conflicts with PHP 8.2

### LINKED ISSUE #13: Fix PHP 8.2 deprecations
We need to refactor the code to fix the following PHP 8.2 deprecation warning:

```
Deprecated: Creation of dynamic property ImprovedExternalProducts::$orderUtil is deprecated in \example.com\wp-content\plugins\woocommerce-improved-external-products\wc-improved-external-products.php on line 214
```

### PR DESCRIPTION
**Pull request by brualio**
Closes #13

### CODE REVIEW

**Review by BrunoPavlinic98 — APPROVED** [review]
LGTM

File: wc-improved-external-products.php [hunk 1]
```diff
@@ -17,6 +17,7 @@ class ImprovedExternalProducts {
 	
 	protected $plugin_version = '1.6.2';
 	public    $settings;
+	public    $orderUtil;
```
Comment (MohamadNateqi) [inline wc-improved-external-products.php]: WPCS - camel case is not allowed.

Same about type hint.

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Review by YordanSoares — APPROVED** [review]
LGTM!

**Review by alexmigf — APPROVED** [review]
LGTM!

**Pull request merged**